css/includes/components/_global-menu.scss

```diff
@@ -37,6 +37,7 @@
 .topbar {
     background-color: $mainmenu-bg;
     color: $mainmenu-fg;
+    z-index: 2000;
```

Comment:
You should use bootstrap variable for z-indexes : https://getbootstrap.com/docs/5.0/layout/z-index/

Test to be sure, but i think we need fixed or sticky var